src/main/scala/com/hs/pipeline/demo/transformations/ContractTransformation.scala

```diff
@@ -14,8 +14,9 @@ object ContractTransformation {
     val transformations: List[DataFrame => DataFrame] = List(
       Clients.joinWithClient(clientDf),
       Country.joinWithCountry(countryDf),
-      calculateParticipation,
-      selectBvColumns
```

Comment:
I think u didn't rebase your branch on master, why is the "**selectColumns**" transformation missing ?
try to fix that, retest and recreate the pull request